_data/toc.yaml

```diff
@@ -1583,6 +1577,52 @@ manuals:
           title: Install Buildx
         - path: /build/buildx/multiple-builders/
           title: Using multiple builders
+    - sectiontitle: Continuous integration
+      section:
+        - path: /build/ci/
+          title: CI with Docker
+        - sectiontitle: GitHub Actions
+          section:
+          - path: /build/ci/github-actions/
+            title: Docker GitHub Actions
+          - path: /build/ci/github-actions/gha-tutorial/
+            title: Tutorial
```

Comment:
IMO we can flatten this out a bit - the `CI with Docker` page could become the section overview.

I'm not sure if we need to next the github-actions into a separate section?

Reply:
Yeah currently we don't have information on any other CI platforms... 😅 but I did it this way in anticipation that we _might_ expand with other examples later. I could flatten it now, and then we can restructure at a later point if/when we have more to say?

Reply:
Ah no, I see :smile: That makes sense, happy to keep it as it :tada: